aws/amazon-ecs-cli-v2
# Share the same names for application template and configuration files

The name format is shared so that we don't duplicate it in 2 places.


Closes #294.

### Manual test
Successfully deployed the generated pipeline CFN template in my personal account

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
